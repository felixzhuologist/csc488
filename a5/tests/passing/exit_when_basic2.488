{
  var c : Integer
  c := 0
  while c < 3 do {
    write c, newline
    exit when false
    c := c + 1
  }
  write "after", newline
}