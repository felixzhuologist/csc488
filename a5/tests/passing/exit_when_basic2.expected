0
1
2
after
